example-context/example-service/example-it/src/test/resources/wildfly-config/standalone-single.xml

```diff
@@ -442,6 +442,7 @@
                 <simple name="java:global/eventlog.insertion.strategy" value="uk.gov.justice.services.eventsourcing.repository.jdbc.AnsiSQLEventLogInsertionStrategy" type="java.lang.String"/>
                 <simple name="java:global/envelope.validation.exception.handler" value="uk.gov.justice.services.core.envelope.RethrowingValidationExceptionHandler" type="java.lang.String"/>
                 <simple name="java:global/fileservice.metadata.sql.provider.strategy" value="uk.gov.justice.services.fileservice.repository.AnsiMetadataSqlProvider" type="java.lang.String"/>
+                <simple name="java:global/file.system.url.resolver.strategy" value="uk.gov.justice.services.core.json.VfsUrlResolverStrategy" type="java.lang.String"/>
```

Comment:
Is this needed if we're using @Alternative?

Reply:
No, I will remove